.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,6 @@
 <!--
-Thank you for helping us improve the Azure IoT Node.js SDK!
+Hi there!
+iothub-explorer will be retired on November 31st, 2018 and has been replaced by the [Azure CLI IoT Extension](https://aka.ms/iotcli).
 
-Here's a little checklist of things that will help it make its way to the repository: Note that you don't have to check all the boxes, we can help you with that. 
-This being said, the more you do, the quicker it'll go through our gated build! 
---> 
-
-# Checklist
-- [ ] I have read the [contribution guidelines] (https://github.com/Azure/iothub-explorer/blob/master/.github/CONTRIBUTING.md).
-
-# Reference/Link to the issue solved with this PR (if any)
-
-# Description of the problem
-<!-- Please be as precise as possible: what issue you experienced, how often... -->
-
-# Description of the solution
-<!-- How you solved the issue and the other things you considered and maybe rejected --> 
+We will not address any pull-request regarding iothub-explorer except those related to the migration to the azure cli.
```

Comment:
Capitalize Azure CLI

Reply:
consider replacing text with 'We will not address any pull-request against iothub-explorer except for those related to migrating towards the Azure CLI.'